cloud_edition/flexibility_mode/docs/crontasks.rst

```diff
@@ -64,6 +64,7 @@ If you don't want to use this wrapper you can prepend `SHELL=/bin/bash`, for exa
     20 0 * * * pimee:project:recalculate
     0 2 * * * pimee:project:notify-before-due-date
     0 23 * * * akeneo:reference-entity:refresh-records --all
```

Comment:
Arg `--all` has been removed during this change: https://github.com/akeneo/pim-enterprise-dev/pull/10562
```suggestion
    0 23 * * * akeneo:reference-entity:refresh-records 
```